samples/openapi3/server/petstore/springboot-nullable-disabled/src/main/java/org/openapitools/model/Category.java

```diff
@@ -0,0 +1,112 @@
+package org.openapitools.model;
+
+import java.net.URI;
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.time.OffsetDateTime;
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+
+import java.util.*;
+import javax.annotation.Generated;
+
+/**
+ * A category for a pet
+ */
+
+@Schema(name = "Category", description = "A category for a pet")
+@Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
+public class Category   {
+
+  @JsonProperty("id")
+  private Long id;
+
+  @JsonProperty("name")
+  private String name;
+
+  public Category id(Long id) {
+    this.id = id;
+    return this;
+  }
+
+  /**
+   * Get id
+   * @return id
+  */
+  
+  // Rely on the @JsonProperty annotation on the variable and ignore the getter methods.
+  @JsonIgnore
+  @Schema(name = "id", required = false)
+  public Optional<Long> getId() {
+    return Optional.ofNullable(id);
+  }
+
+  public void setId(Long id) {
+    this.id = id;
+  }
+
+  public Category name(String name) {
+    this.name = name;
+    return this;
+  }
+
+  /**
+   * Get name
+   * @return name
+  */
+  @Pattern(regexp = "^[a-zA-Z0-9]+[a-zA-Z0-9\\.\\-_]*[a-zA-Z0-9]+$") 
+  // Rely on the @JsonProperty annotation on the variable and ignore the getter methods.
+  @JsonIgnore
```

Comment:
Not sure why you need the ignore. IMHO this is also working without the annotation

Reply:
I haven't made the changes to use the JDK8 module yet - once I do I think I can also remove these annotations. 

This is for non-JDK8 support, but since the Spring project is JDK8+ only now - I'll make that change. Should have time later this week.